docs/workforce-management-framework/topics/ref-introducing-raincatcher.adoc

```diff
@@ -0,0 +1,57 @@
+[id='ref-introducing-raincatcher-{chapter}']
+= Defining {WFM-RC-NameShort} Areas of Focus
+
+.Areas of Focus
+|===
+|Area |Focus
+
+|Framework
+|A set of modules for building mobile and website Workforce Management solutions
```

Comment:
`Generic and developer friendly framework - set of classes for building` 
`mobile and web`